### PR TITLE
W-14208959-ACB-non-tut-address-PM-comments

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

File: modules/ROOT/pages/int-create-integrations.adoc [hunk 1]
```diff
@@ -157,10 +149,33 @@ In addition to the xref:acb-components.adoc#builtin-connectors[built-in connecto
 you can download many other connectors from Anypoint Exchange.
 
 
-To import a connector from Exchange:
+To import a connector from Exchange and add it to your code:
```
Comment (dukesphere) [inline modules/ROOT/pages/int-create-integrations.adoc]: Should we use "code" here? Maybe to their "configuration"?

Comment (hannanelson) [inline modules/ROOT/pages/int-create-integrations.adoc]:
```suggestion
To import a connector from Exchange and add it to your configuration:
```

File: modules/ROOT/pages/int-create-integrations.adoc [hunk 2]
```diff
@@ -157,10 +149,33 @@ In addition to the xref:acb-components.adoc#builtin-connectors[built-in connecto
...
 
 
-To import a connector from Exchange:
+To import a connector from Exchange and add it to your configuration:
 
 . In the Explorer view, open the configuration XML file for your project, such as `my-project-name.xml`.
 . Click the image:icon-tree-view.png["",15,15] (*Show Mule graphical mode*) icon in the activity bar to open the UI canvas if it doesn't open automatically.
+
+. Add the connector the same way you added other components from the canvas:
+.. In the canvas, click the image:icon-plus.png["",15,15] (*Add component*) icon.
```
Comment (dukesphere) [inline modules/ROOT/pages/int-create-integrations.adoc:156]: Do you need png alt text?

File: modules/ROOT/pages/int-create-integrations.adoc [hunk 3]
```diff
@@ -157,10 +149,33 @@ In addition to the xref:acb-components.adoc#builtin-connectors[built-in connecto
...
+.. In the *Add Component* panel, click *Connectors*.
+.. Click the connector name and then click the operation to add, such as *Publish*:
++
+image::int-add-connector-operation.png["Add Publish operation from the Anypoint MQ Connector"]
++
+If the connector is not available locally, click the image:icon-search-exchange.png["",15,15] (*Search in Exchange*) toggle:
```
Comment (dukesphere) [inline modules/ROOT/pages/int-create-integrations.adoc:162]: I'm guessing not (alt text for png). ¯\_(ツ)_/¯

File: modules/ROOT/pages/tut-af-integrate-use-dataweave.adoc [hunk 4]
```diff
@@ -148,7 +148,7 @@ The user snippets provide starting configurations for several Mule components, i
 +
 Below the Listener, click the *+* icon, and navigate to *Snippets* > *User Snippets* > *Transform Message*.
 +
-If the *User Snippets* pane is empty at first, wait a moment, and try again. Alternatively, navigate to the `muledx:transform-message` snippet from the configuration XML.
+If the *User Snippets* panel is empty at first, wait a moment, and try again. Alternatively, navigate to the `muledx:transform-message` snippet from the configuration XML.
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-integrate-use-dataweave.adoc:151]: is this the snippet that ships with the IDE?

Comment (dukesphere) [reply modules/ROOT/pages/tut-af-integrate-use-dataweave.adoc:151]: Built-in transform message seems to be this one:

```
    <ee:transform doc:name="" doc:id="ixszhb" >
      <ee:message >
        <ee:set-payload doc:name="Set payload" doc:id="nglfow" ><![CDATA[%dw 2.0
    output application/json
    ---
    {
      firstName : "",
      lastName : "",
    }]]></ee:set-payload>
      </ee:message>
    </ee:transform>
```

**Review by dukesphere — APPROVED** [review]
alt text for the pngs if needed here

File: modules/ROOT/pages/int-create-integrations.adoc (hunk 3 above)
Comment (hannanelson) [reply modules/ROOT/pages/int-create-integrations.adoc:162]: You don't need alt text for decorative images. Think about how a screen reader would read this.

File: modules/ROOT/pages/int-create-integrations.adoc (hunk 2 above)
Comment (hannanelson) [reply modules/ROOT/pages/int-create-integrations.adoc:156]: Not for decorative (icons) images.

**Pull request merged**